TipCatDotNet.Api/Models/HospitalityFacilities/FacilityResponse.cs

```diff
@@ -1,16 +1,20 @@
+using System.Collections.Generic;
+
 namespace TipCatDotNet.Api.Models.HospitalityFacilities
 {
     public class FacilityResponse
     {
-        public FacilityResponse(int id, string name, int accountId)
+        public FacilityResponse(int id, string name, int accountId, List<MemberResponse>? members)
```

Comment:
Couldn't be null, I guess

Reply:
We have an endpoint that creates an empty facility. Therefore this new facility can stands empty for a long time.

Reply:
Yes, of course, but look at that from another angle: no members is zero members, on the other hand, zero not equals to null. So an empty list is suitable here and much more safe in terms of NRE

Reply:
It makes sense, i'll fix it